build.gradle.kts

```diff
@@ -0,0 +1,41 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    kotlin("jvm").apply(false)
+}
+
+
+allprojects {
+    group = "playground"
+
+    repositories {
+        mavenLocal()
+        mavenCentral()
+        google()
+        jcenter()
+        maven("https://dl.bintray.com/kotlin/kotlin-eap/")
+        maven("https://kotlin.bintray.com/kotlinx/")
+    }
+
+    tasks.withType<KotlinCompile> {
+        kotlinOptions.jvmTarget = "1.8"
+        kotlinOptions.freeCompilerArgs = listOf(
```

Comment:
https://youtrack.jetbrains.com/issue/KT-41985

Please, fix this to not overwrite other arguments.